src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.html

```diff
@@ -17,27 +17,25 @@
        routerLink="/services">
        Cluster->Services</a>
   </cd-alert-panel>
-  <span *ngIf="!showMigrateAndReplicationActions; else migrateAndReplicationActionTpl">
-    <cd-table-actions class="btn-group mb-4 me-2"
-                      [permission]="permission"
-                      [selection]="selection"
-                      [tableActions]="createTableActions"
-                      [primaryDropDown]="true">
-    </cd-table-actions>
-  </span>
-  <ng-template #migrateAndReplicationActionTpl>
-    <cd-table-actions class="btn-group mb-4 me-2"
-                      [permission]="permission"
-                      [selection]="selection"
-                      [tableActions]="multisiteReplicationActions">
-    </cd-table-actions>
-    <cd-table-actions class="btn-group mb-4 me-2 secondary"
-                      [permission]="permission"
-                      [btnColor]="'light'"
-                      [selection]="selection"
-                      [tableActions]="migrateTableAction">
-    </cd-table-actions>
-  </ng-template>
+  <cd-table-actions class="btn-group mb-4 me-2"
+                    [permission]="permission"
+                    [selection]="selection"
+                    [tableActions]="multisiteReplicationActions">
+  </cd-table-actions>
+  <cd-table-actions *ngIf="showMigrateAndReplicationActions"
+                    class="btn-group mb-4 me-2 secondary"
+                    [permission]="permission"
+                    [btnColor]="'light'"
+                    [selection]="selection"
+                    [tableActions]="migrateTableAction">
+  </cd-table-actions>
+  <cd-table-actions *ngIf="!showMigrateAndReplicationActions"
+                    class="btn-group mb-4 me-2"
+                    [permission]="permission"
+                    [selection]="selection"
+                    [tableActions]="createTableActions"
+                    [primaryDropDown]="true">
+  </cd-table-actions>
```

Comment:
is it intentional to show multisite setup button and create realm button together?

Reply:
Yes, these buttons can co-exist, you can run through the wizard to create realm/zg/zone at once on single click. If a user wants to create a new realm/zg/zone individually that can be done as well.